docs/source/rule-staging.rst

```diff
@@ -0,0 +1,109 @@
+Rule Staging
+============
+
+Rule staging allows dynamic toggling of rules from a staging to production state, and vice versa.
+Staged rules will only send alerts for historical retention, and the alerts will not be sent to any
+user-defined outputs, such as Slack, PagerDuty, etc.
+
+Enabling Rule Staging
+---------------------
+
+By default, the rule staging feature is not enabled. It can be enabled with the following command:
+
+.. code-block:: bash
+
+  python manage.py rule-staging enable --true
+
+The change will be reflected in the ``conf/globals.json`` file:
+
+.. code-block:: json
+  :caption: ``conf/global.json``
+
+  {
+    "infrastructure": {
+      "rule_staging": {
+        "cache_refresh_minutes": 10,
+        "enabled": true,
+        "table": {
+          "read_capacity": 20,
+          "write_capacity": 5
+        }
+      }
+    }
+  }
+
+Configuration Options
+---------------------
+
+A few configuration options are available to customize the feature to your needs.
+
+===========================  ========  =======  ===========
+Key                          Required  Default  Description
+---------------------------  --------  -------  -----------
+``cache_refresh_minutes``    ``No``    ``10``   Maximum amount of time (in minutes) the Rule Processor
+                                                should wait to force refresh the rule staging information.
+``table.read_capacity``      ``Yes``   ``20``   DynamoDB read capacity to allocate to the table that stores staging
+                                                information. The default setting should be sufficient in most use cases.
+``table.write_capacity``     ``Yes``   ``5``    DynamoDB write capacity to allocate to the table that stores staging
+                                                information. The default setting should be sufficient in most use cases.
+===========================  ========  =======  ===========
```

Comment:
I've actually been getting rid of the `Required` column. If a value has a default, then it isn't required

Reply:
@austinbyers I think historically 'default' has actually meant "there is a default in the terraform generation code, and you can omit the config option entirely"

Reply:
but I don't actually like the 'required' column anyway tbh so will nix as well